community/contributing/docs/github-changes.md

```diff
@@ -28,13 +28,13 @@ sidebar:
 
   For more information, see [commit message guidelines](https://www.spinnaker.io/community/contributing/submitting/#commit-message-conventions).
 
-  In the **second field**, provide a clear and detailed description of your PR. Do not to leave these feilds blank, as it is helpful to reviewers when merging your request to have additional context as to what your pull request is about.
+  In the **second field**, provide a clear and detailed description of your PR. Do not to leave these fields blank, as it is helpful to reviewers when merging your request to have additional context as to what your pull request is about.
```

Comment:
These "first field," "second field" don't work to well either, but that's how it was before this PR, so ok for now. (Heck, I probably wrote that myself 🌵 )

Reply:
@dorbin A new contributor submitted this PR during Spinnaker Gardening Days. After reading your comments, I followed the instructions, and then reordered. Also ensured field names matched the UI. Moved the PR title checker content to the `Open a pull request` step because GitHub only checks the PR title, not the title of the commit. Added two images for clarity.

Reply:
👍